build.gradle.kts

```diff
@@ -93,6 +93,11 @@ dependencies {
         because("Lowest version that works with our dependencies")
     }
 
+    // Higher versions fail in our tests with ClassNotFoundException during SCM initialization unless Jenkins is updated
+    "optionalPluginImplementation"("org.jenkins-ci.plugins:git:4.9.4") {
```

Comment:
So with this version, you don't need to update the required Jenkins version?

Reply:
Right, this one works fine in tests